### DISCUSSION TITLE
Node 16 LTS

### DISCUSSION THREAD

**Discussion by silvioprog** [deta/cloud-docs#516] [Ideas]
Hey Deta team.

First of all, thanks a lot for this amazing service! 🤗

So, I have a naive suggestion. Please let me know if this is the right place to ask. But, what do you think about using [Node 14.x/16.x LTS](https://medium.com/the-node-js-collection/node-js-16-available-now-7f5099a97e70) instead of 12.x/14.x?

Please take a look at this table below:

https://endoflife.date/nodejs

As shown there, Node 12.x ended Security Support since 5 months ago (30 Apr 2022), while 14.x/16.x still active.

TIA

**Comment by AyushSehrawat** [comment]
Apply for Deta space! It has multiple language support and frameworks... https://alpha.deta.space

**Reply by AyushSehrawat** [reply]
For the time being uss 14

**Comment by silvioprog** [comment]
I have been trying to access https://alpha.deta.space, but I don't have access. 🥲

**Reply by AyushSehrawat** [reply]
You have to apply and be in the waitlist! Make some good projects by using deta and get access faster :).

**Reply by silvioprog** [reply]
Yay, perfect! :)

**Reply by silvioprog** [reply]
Just curious, will [alpha.deta.space](https://alpha.deta.space) discontinue [deta.sh](https://www.deta.sh)? 🤔

**Reply by abdelhai** [reply]
we will not discontinue deta cloud (deta.sh), but we will move everything to space at some point. it's should be a few clicks tho (or no clicks for many apps). deta space does what deta cloud does and more
